src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,174 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    public MyLinkedList() {
+        head = null;
+        tail = null;
+        size = 0;
+    }
+
+    public MyLinkedList(T element) {
+        Node<T> firstNode = new Node<>(element);
+        head = tail = firstNode;
+        size = 1;
+    }
+
+    private static class Node<T1> {
+        T1 storedItem;
+        Node<T1> previous;
+        Node<T1> next;
+
+        public Node(T1 element) {
+            storedItem = element;
+            previous = null;
+            next = null;
+        }
+    }
+
+    public void checkBounds(int index) {
```

Comment:
what`s wrong with access modifiers of all helper methods?

Reply:
done